### ISSUE TITLE
yank analyze extract-trajectory gives error in 0.23.1

### ISSUE THREAD

**Issue by sgill2** [choderalab/yank#1048]
Is there be supposed to be a difference in between getting the trajectories in 0.23.1 versus 0.22.3? I was trying to extract the trajectory of a SAMS simulation using version 0.23.1 using `yank analyze extract-trajectory --netcdf complex.nc --trajectory output.dcd --replica 0` and I get the following error message:

```
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/bin/yank", line 11, in <module>
    load_entry_point('yank==0.23.1', 'console_scripts', 'yank')()
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/cli.py", line 73, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/commands/analyze.py", line 149, in dispatch
    single_run()
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/mpi.py", line 263, in _wrapper
    return run_single_node(rank, task, *args, **kwargs)
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/mpi.py", line 212, in run_single_node
    result = task(*args, **kwargs)
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/commands/analyze.py", line 144, in single_run
    output = analyze.analyze_directory(args['--store'], **analyzer_kwargs)
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/analyze.py", line 767, in analyze_directory
    auto_experiment_analyzer = ExperimentAnalyzer(source_directory, **analyzer_kwargs)
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/site-packages/yank/analyze.py", line 293, in __init__
    analysis_script_path = os.path.join(store_directory, 'analysis.yaml')
  File "/hpc/apps/2018/anaconda3/v5.0.1.app/envs/yank_v0.23.1/lib/python3.6/posixpath.py", line 80, in join
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```
Using the exact same command with version 0.22.3 (on a different, older experiment) works.
From the error message it looks like the `store_directory` variable isn't properly being assigned a value.

If you're having trouble reproducing this, I'd be happy to provide an example case.

**Comment by Lnaden** [comment]
Try `--netcdf=complex.nc` and `--trajectory=output.dcd` instead.

**Comment by Lnaden** [comment]
The Docopt parser is powerful, but also a bit flakey if you have some small things out of place.

**Comment by sgill2** [comment]
I'm still getting the same error message with both
`yank analyze extract-trajectory --netcdf=complex.nc --trajectory=output.dcd --replica 0`
and
`yank analyze extract-trajectory --netcdf=complex.nc --trajectory=output.dcd --replica=0`

**Comment by Lnaden** [comment]
So. I may or may not have accidentally deleted the single if statement which actually executes the `extract-trajectory` code when I did the multi-analysis stuff. Not saying I did, but If I did, its a quick 2-line fix with an `if` statement

If time is of the essence, I can tell you which file to edit to fix it, otherwise I'll get a release cut today.

And just to dispel any ambiguity in the first paragraph: I did. Sorry!
